config/earlybird.json

```diff
@@ -42,13 +16,61 @@
     {
       "level_name": "low",
       "level_id": 4
+    },
+    {
+      "level_name": "info",
+      "level_id": 5
     }
   ],
-  "text_ignore_patterns": ["EARLYBIRD-IGNORE","EB-IGNORE"],
-  "filename_skip_text_scanning_extensions": [".pem",".p12",".pkcs12",".jks",".ppk",".pfx",".asc",".private_key",".keychain",".keystore",".keyring",".pcap",".htpasswd",".kwallet",".tblk",".min.js",".js.map",".css",".min.css",".woff",".ttf",".otf",".png",".gif",".jpg",".jpeg",".svg",".tiff"],
+  "text_ignore_patterns": ["EARLYBIRD-IGNORE", "EB-IGNORE"],
+  "filename_skip_text_scanning_extensions": [
+    ".pem",
+    ".p12",
+    ".pkcs12",
+    ".jks",
+    ".ppk",
+    ".pfx",
+    ".asc",
+    ".private_key",
+    ".keychain",
+    ".keystore",
+    ".keyring",
+    ".pcap",
+    ".htpasswd",
+    ".kwallet",
+    ".tblk",
+    ".min.js",
+    ".js.map",
+    ".css",
+    ".min.css",
+    ".woff",
+    ".ttf",
+    ".otf",
+    ".png",
+    ".gif",
+    ".jpg",
+    ".jpeg",
+    ".svg",
+    ".tiff"
+  ],
   "fail_threshold_level": 2,
   "display_threshold_level": 3,
   "display_confidence_threshold_level": 2,
-  "earlybird_config_url": "https://raw.githubusercontent.com/americanexpress/earlybird/master/config/earlybird.json",
-  "version": "1.25.0"
-}
\ No newline at end of file
+  "earlybird_config_url": "https://raw.githubusercontent.com/americanexpress/earlybird/main/config/earlybird.json",
+  "version": "2.0.0",
+  "name": "earlybird",
+  "adjusted_severity_categories_patterns": [
```

Comment:
we should maybe document this but not as part of our default configuration

Reply:
It's documented in the readme